### ISSUE TITLE
Skip minimap when pressing Ctrl+W W

### ISSUE THREAD

**Issue by Akselmo** [wfxr/minimap.vim#164]
## Check list

- [X] I have read through the [README](https://github.com/wfxr/minimap.vim/blob/master/README.md) (especially F.A.Q section)
- [X] I have searched through the existing issues

## Environment info

- OS
    - [X] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Others:

## Version info

NVIM v0.7.2
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3

## Question / Problem and steps to reproduce

Basically I'd like to be able to skip the minimap completely when pressing `Ctrl+w w`. I just dont know how. I guess this could be done with vimscript?

**Comment by ZNielsen** [comment]
Would #146 accomplish this, or is that too aggressive?

**Comment by Akselmo** [comment]
> Would #146 accomplish this, or is that too aggressive?

That would also work
